app/javascript/components/ansible-playbook-edit-catalog-form/schema.js

```diff
@@ -0,0 +1,141 @@
+import { componentTypes } from '@@ddf';
+
+function transformOptions(options) {
+    return options.map(([label, value]) => ({ label, value }));
+}
+
+function transformCurrencies(currencies) {
+    return currencies.map((currency) => {
+      return {
+        label: `${currency.symbol} ${currency.full_name}`,
+        value: currency.id,
+      };
+    });
+}
+
+const createSchema = ({available_catalogs, zones, currencies}) => {
```

Comment:
if this becomes too large, lates move each tab item schema into different file

Reply:
I have restructured to 3 separate functions for each tab in same file. Do you want me to have each function schema in separate file ??

Reply:
this is fine for now..